### ISSUE TITLE
Gut microbiome profiling of a rural and urban South African cohort reveals biomarkers of a population in lifestyle transition

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#329]
Gut microbiome profiling of a rural and urban South African cohort reveals biomarkers of a population in lifestyle transition – O. H. Oduaran – BMC Microbiology
https://bmcmicrobiol.biomedcentral.com/articles/10.1186/s12866-020-02017-w

**Comment by Lonercode** [comment]
Hello @SvetlanaUP I would like to curate this please.
Thanks

**Comment by nityasinghal-14** [comment]
Hello @Lonercode, I see you have been assigned this curation. Would you be open to collaborating? 
I'm looking forward to it, thank you.

**Comment by omojokunoluwatomisin10** [comment]
Hi @Lonercode, are you still working on your curation article or do you need help, if yes let's collaborate on the work

**Comment by SvetlanaUP** [comment]
@Lonercode is https://bugsigdb.org/Study_1057 ready for review?

**Comment by RotimiDesire** [comment]
Hi @SvetlanaUP 

Since there has been no response from the original curator, may I go ahead to wrap this up?

**Comment by SvetlanaUP** [comment]
@RotimiDesire ok

**Comment by Lonercode** [comment]
Hi @SvetlanaUP I was finished with this work but had not put it up for review. A very delicate and personal matter came up in the last week that made it impossible for me to continue in the time. I couldn't even get service. I would like to know if I can proceed on my work.

**Comment by SvetlanaUP** [comment]
Sure! @RotimiDesire @Lonercode continue in collaboration.

**Comment by RotimiDesire** [comment]
Okay.

**Comment by RotimiDesire** [comment]
Hi @SvetlanaUP 

The study is ready for review now. 
https://bugsigdb.org/Study_1057

**Comment by Scholarpat** [comment]
Hello @Lonercode @RotimiDesire,

This was a very tasking curation, and you both did a good job curating important details. Well done. I've made a few adjustments below:

- Sample sizes were left out for experiments 2 to 6. I found these within the text and have added them.
- I've edited the signatures to include only those taxa that met the significant threshold of 0.05.

This curation is complete @SvetlanaUP.

Thank you!
